schemas/com.apple.swiftui/immersive_space/jsonschema/1-0-0

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self": {
+    "vendor": "com.apple.swiftui",
+    "name": "immersive_space",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "description": "Schema for an immersive space entity (VisionOS)",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "format": "uuid",
+      "description": "Randomly generated ID for the view of the immersive space. Generated by the tracker if not provided by the user."
+    },
+    "immersive_space_id": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "The identifier of the immersive space to present."
+    },
+    "immersion_style": {
+      "type": "string",
```

Comment:
This should be nullable since it is not required. To do that, you'll need to add null to both the type and the enum values, [see here](https://github.com/snowplow/iglu-central/blob/f89b6aea3231abb49fb86e87f999b66f9131db3e/schemas/com.snowplowanalytics.snowplow/mobile_context/jsonschema/1-0-2#L72-L73)

Reply:
I see that some nullable enums have null in their values list, but I don't think it is required? Change_form and focus_form don't

Reply:
I think in the newer schemas, we tend to add `null` in the values list (see [the media player](https://github.com/snowplow/iglu-central/commit/25d59d3e39bb54b8c9b431a3c435f8ea4a15b5f8#diff-1b5edede2341bc080382bc706d0f53d6d37514754c8059496a2fd118bb4c7e1eR60-R70)). I remember Ian telling me that it was required for some reason, but can't find the conversation, you can double check with him to be sure.